zebra-network/src/meta_addr/tests/prop.rs

```diff
@@ -339,7 +343,11 @@ proptest! {
             None
         };
 
-        let address_book = Arc::new(std::sync::Mutex::new(AddressBook::new_with_addrs(&Config::default(), Span::none(), addrs)));
+        let address_book = Arc::new(std::sync::Mutex::new(AddressBook::new_with_addrs(
+            SocketAddr::from_str("0.0.0.0:0").unwrap(),
```

Comment:
Just in case you might want to avoid the `unwrap`:

```suggestion
            SocketAddr::from(([0,0,0,0], 0)),
```

Reply:
I think the string is clearer - it's not obvious that the second address is an IPv4 address and port.